### ISSUE TITLE
batchOperations.js error

### ISSUE THREAD

**Issue by andreys70** [awslabs/aws-lambda-redshift-loader#224]
Since batch structure for items has changed in 2.7.9 reprocessBatch() needs to be revised. It is currently produces an error when trying to reprocess the batch. The "data.entries.SS" call is an issue.

**Comment by IanMeyers** [comment]
Doh! Looking

**Comment by IanMeyers** [comment]
Have a go with 25ca772?

**Comment by andreys70** [comment]
This can't work (I just ran a test to double-check) as "data.entries.SS" is not valid structure:
```
/aws-lambda-redshift-loader/node_modules/aws-sdk/lib/request.js:31
            throw err;
            ^

TypeError: Cannot read property 'SS' of undefined
    at /aws-lambda-redshift-loader/batchOperations.js:284:35
```

**Comment by andreys70** [comment]
Also, I removed all legacy support code, to eliminate above error. Now I'm getting this:
```
/aws-lambda-redshift-loader/node_modules/aws-sdk/lib/request.js:31
            throw err;
            ^

TypeError: Cannot read property 'S' of undefined
    at /aws-lambda-redshift-loader/batchOperations.js:323:69
    at Array.map (<anonymous>)
    at /aws-lambda-redshift-loader/batchOperations.js:322:53
```
lines 322 and 323 now:
```
322                                    data.entryMap.L.map(function (item) {
323                                       processFiles.push(item.file.S);
```

**Comment by andreys70** [comment]
I get it working with removed legacy support code, and changed "item.file.S" references to "item.M.file.S"

**Comment by IanMeyers** [comment]
Sorry about that. These are fixed in 660a291.

**Comment by andreys70** [comment]
Tested. All works now. Thank you!
